### DISCUSSION TITLE
FeatureRequest: Diagnostics by lsp be pastable

### DISCUSSION THREAD

**Discussion by roccho-dev** [helix-editor/helix#11937] [Ideas]
I want to control the diagnostic results that LSP tells me in the buffer, which can be done manually.
As for use cases, in my case it is mainly because I want to use the information in the buffer to talk to the so called generative AI tool.

What all editors will devise in the future is about this linkage and ux. I think Helix should be useful because of its compactness, but also because it opens the way to an already existing extension, LSP.

**Comment by wldmr** [comment]
So you want to copy the diagnostic into another tool? `:yank-diagnostic` does that.
